legacy/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../.babelrc"
```

Comment:
For what Babel is used for?

Reply:
Jest can't work with Vue modules without Babel